foundation/foundation-upgrade/src/test/resources/test-files/v1_7_0/EnableHabushuBuildCacheMigration/skip-migration/existing-build-directory/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
S: similarly, I'd align these folder names to be more self explanatory of the requirement vice outcome

Reply:
Done!